src/components/Button/index.js

```diff
@@ -69,8 +69,10 @@ const StyledButton = styled(({
   )
 })`${props => styleMap(props.size)}`
 
+StyledButton.displayName = 'Button'
```

Comment:
`displayName` lets you override the default debugging name that JSX creates, which in the case of a `styled-component` is just `Styled(Component)` so overriding here makes debugging easier